src/google/protobuf/io/tokenizer.h

```diff
@@ -322,6 +337,10 @@ class PROTOBUF_EXPORT Tokenizer {
   // Read a new buffer from the input.
   void Refresh();
 
+  // Check the most recent character read to verify that the input
+  // is correctly encoded UTF8. Does nothing if !require_valid_utf8_.
+  void CheckUtf8();
```

Comment:
The naming style for this type of call that is controlled by an option seems to be: MaybeCheckUtf8